dialob-components/dialob-groovy-functions/src/main/java/io/dialob/groovy/DialobGroovyFunctionsAutoConfiguration.java

```diff
@@ -27,10 +29,11 @@
 public class DialobGroovyFunctionsAutoConfiguration {
 
   @Bean
-  public GroovyFunctionRegistry groovyFunctionRegistry(ApplicationContext applicationContext, FunctionRegistry functionRegistry) {
+  public GroovyFunctionRegistry groovyFunctionRegistry(ApplicationContext applicationContext, FunctionRegistry functionRegistry, DialobSettings dialobSettings) {
     GroovyFunctionRegistry groovyFunctionRegistry = new GroovyFunctionRegistry(applicationContext, functionRegistry);
     List<String> functions = new ArrayList<>();
     functions.add("classpath:/scripts/FoaasService.groovy");
```

Comment:
This have been an example. Remove FoaasService...

Reply:
This example function removed